config/env/default.js

```diff
@@ -10,5 +10,7 @@ module.exports = {
 	port: process.env.PORT || 3000,
 	templateEngine: 'swig',
 	sessionSecret: 'MEAN',
-	sessionCollection: 'sessions'
+	sessionCollection: 'sessions',
+	logo: 'modules/core/img/brand/logo.png',
+	favicon: '/modules/core/img/brand/favicon.ico'
```

Comment:
why is one file relative and the other absolute?  I know that you just took the value and put it in the config, but should this be switched to a relative path?  Templates follow relative paths.